CONTRIBUTING.md

```diff
@@ -22,7 +22,8 @@ From a separate terminal:
 - `flutter run`
 
 From the packages/devtools directory:
-- `pub global activate webdev`
+- `pub global activate webdev` (install webdev globally)
```

Comment:
nit: when writing instructions like this make it is nice for a user to be able to just copy and paste the lines and have everything work instead of pasting one line at a time.
For example
echo "install webdev globally"
or alternate formatting so the comments don't mess with the expressions.